src/board/Cell.jsx

```diff
@@ -1,9 +1,26 @@
+import React, { useState } from "react";
 import "./Cell.css";
 import PropTypes from "prop-types";
+import { isValidMove } from "../validation/validateMove";
+
+function Cell({ cellSpot, handleClick, gameStatus }) {
+  console.log(cellSpot);
+  const [isFlipped, setIsFlipped] = useState(false);
+
+  const handleCellClick = () => {
+    const audio = new Audio(process.env.PUBLIC_URL + "/cell.wav");
```

Comment:
One enhancement is that you can keep the OI stuff outside of this component. So having a function for audio that does the work on the edge of your app and then calling the function in your component makes sense from a decoupling standpoint.

Reply:
oh true! thank you!